Cargo.toml

```diff
@@ -1,12 +1,14 @@
 [package]
-name = "rust-app"
-version = "0.2.1"
-authors = ["yhql"]
+name = "app-boilerplate-rust"
+version = "1.0.0"
+authors = ["yhql", "agrojean-ledger"]
 edition = "2021"
 
 [dependencies]
 nanos_sdk = { git = "https://github.com/LedgerHQ/ledger-nanos-sdk.git" }
 nanos_ui = { git = "https://github.com/LedgerHQ/ledger-nanos-ui.git" }
+include_gif = { git = "https://github.com/LedgerHQ/sdk_include_gif" }
```

Comment:
So that I can use the crab gif logo on the main menu. Here : https://github.com/LedgerHQ/app-boilerplate-rust/blob/97e3e2e09e48194d88369a837e31ba058a6e156b/src/app_ui/menu.rs#L40